### PR TITLE
feat: login, signup pages

### PR DESCRIPTION
**Pull request by ddivyansh18**
- Added login, signup pages to NyceKart
image

### CODE REVIEW

File: auth/login.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+              <span class="material-icons-outlined md-36"> shopping_bag </span>
+              <span class="icon-badge white-bg black-border"> </span>
+            </div>
+          </a>
+        </li>
+      </ul>
+    </nav>
+
+    <div class="page-content-container">
+      <section class="login-container card-shadow column-container white-bg">
```
Comment (manish591) [inline auth/login.html]: Use form element for login container. It is semantically right and gives you more control over sections.

File: auth/login.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        <h1>Login</h1>
+        <div class="login-field-container">
+          <label for="email">Email</label>
+          <input type="email" id="email" />
+        </div>
+        <div class="login-field-container">
+          <label for="password">Password</label>
+          <input type="password" name="" id="password" />
```
Comment (manish591) [inline auth/login.html]: Add a required attribute here.

File: auth/login.html (hunk 1 above)
Comment (ddivyansh18) [reply auth/login.html]: Added form element for both the pages

File: auth/login.html (hunk 2 above)
Comment (ddivyansh18) [reply auth/login.html]: @manish591 Fixed

**Review by surajmore98 — APPROVED** [review]

**Review by manish591 — APPROVED** [review]

**Pull request merged**